service-registry/src/test/java/org/apache/servicecomb/serviceregistry/config/TestServiceRegistryConfig.java

```diff
@@ -30,6 +34,7 @@
   @BeforeClass
   public static void initClass() {
     ArchaiusUtils.resetConfig();
+    ConfigUtil.installDynamicConfig();
```

Comment:
so heavy, why not just invoke ArchaiusUtils.setProperty inside test case.

Reply:
done